tests/integration/targets/eos_acl_interfaces/tasks/cli.yaml

```
@@ -27,3 +27,12 @@
   with_items: '{{ test_items }}'
```

It's not necessary, but we could be clearer about what we intend to test here

```suggestion
    ansible_network_cli_ssh_type: paramiko
  with_items: '{{ test_items }}'
```